bench/benchmarks_view.js

```diff
@@ -1,12 +1,89 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import * as d3 from 'd3';
-import Axis from './lib/axis';
```

Comment:
I kind of wish all the React components weren't defined in one script but 🤷‍♂️

Reply:
I thought it would be nice to put all UI in one file, so that there's clear separation — `lib` only contains logic that gets insider the different benchmarks builds, and `bencharks_view` encapsulates all the view logic. As a follow-up, we could split the view components again in a separate folder (e.g. `view` or `components`).

Reply:
Splitting up the components may be a good opportunity to remove React too. Could be easier to do if the UI is split into smaller, discrete pieces

Reply:
Sounds good! Let's follow up with splitting components in a different PR (don't want to blow up the scope of this one too much).